docs/components/identity/mapping-rules/manage-mapping-rules.md

```diff
@@ -26,17 +24,23 @@ To learn more about the concept of mapping rules, see the [mapping rules documen
 
 <TabItem value="add">
 
-1. Click the **Create a mapping rule** button.
+## Add a mapping rule
+
+1. By default, Identity doesn't have **mapping rules** created.
+
+![mapping-rule-management-tab](../img/mapping-rule-management-tab.png)
+
+2. Click the **Create a mapping rule** button.
 
-2. Fill in the required fields:
+3. Fill in the required fields:
 
 - **Mapping Rule ID**: A unique identifier for the mapping rule.
 - **Mapping Rule name**: A user-friendly name.
 - **Claim name** and **Claim value**: These define the JWT claim rule to match.
 
 ![mapping-rule-add-mapping-rule-modal](../img/mapping-rule-add-mapping-modal.png)
 
-3. Click **Create a mapping rule** to save.
+4. Click **Create a mapping rule** to save.
 
 The new mapping rule will appear in the list.
```

Comment:
I wonder if one should link after this to the place for Assigning them to tenants or assigning them to authorizations?